### ISSUE TITLE
Make input to createHTML, createScript, and createScriptURL optional

### ISSUE THREAD

**Issue by shhnjk** [w3c/webappsec-trusted-types#278]
Current implementation of Chrome requires input argument in createHTML and etc required. However, there are cases where argument isn't required (e.g. just returning static HTML string to be used for innerHTML). Therefore, input argument should be optional.

**Comment by shhnjk** [comment]
See https://source.chromium.org/chromium/chromium/src/+/master:components/about_ui/resources/about_credits.js;l=23

**Comment by koto** [comment]
That particular one can be replaced with `trustedTypes.emptyScript`.

**Comment by shhnjk** [comment]
No, it returns actual script URL above, but I have to still provide an argument.

**Comment by shhnjk** [comment]
Can I get consensus if spec in interested in fixing this issue or not?

**Comment by koto** [comment]
Let's wait for more implementation feedback about that? For now it seems like the default policy (which, when called implicitly, gets always the same kind of arguments) is different than other policies (which could even accept any number of arguments, of arbitrary types, or no arguments at all, like you mentioned), so it might be better to separate their interfaces too.
